### PR TITLE
<DatePicker /> vertical spacing & no outer padding props

### PR DESCRIPTION
**Pull request by poseneror**
### 🔦 Summary
This PR includes 2 changes to the DatePicker component:
1. add the ability to remove outer padding from the Date Picker using `noPadding` prop
2. add the ability to control the space between the weeks in the Date Picker using the overridable style param `WeeksVerticalSpacing`
...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
- 🔬 Change is tested
  - [x] Visual test

### CODE REVIEW

File: packages/wix-ui-tpa/src/components/DatePicker/DatePicker.st.css [hunk 1]
```diff
@@ -128,6 +135,8 @@
     CurrentSelectedDayBGColor: color(fallback(value(SelectedDayBGColor), value(DefaultSelectedDayBGColor)));
     CurrentSelectedDayBorderRadius: fallback(value(SelectedDayBorderRadius),value(DefaultSelectedDayBorderRadius));
     CurrentDisabledDayColor: color(fallback(value(DisabledDayColor), value(DefaultDisabledDayColor)));
+
+    CurrentWeeksVerticalSpacing: applyUnit(fallback(value(WeeksVerticalSpacing),value(DefaultWeeksVerticalSpacing)), px);
```
Comment (sivanharel) [inline packages/wix-ui-tpa/src/components/DatePicker/DatePicker.st.css:139]: You have to import the `applyUnit`. This is the reason that all the tests are failing.

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6856/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6856

File: packages/wix-ui-tpa/src/components/DatePicker/DatePicker.st.css [hunk 2]
```diff
@@ -251,6 +277,10 @@
     color: value(CurrentDisabledDayColor);
 }
 
+.calendar::week:not(:last-child) {
+    margin-bottom: value(CurrentWeeksVerticalSpacing);
+}
+
```
Comment (sivanharel) [inline packages/wix-ui-tpa/src/components/DatePicker/DatePicker.st.css:283]: Can you please move it to be before the css of the day? (Before line 235)

File: packages/wix-ui-tpa/src/components/DatePicker/DatePicker.st.css [hunk 3]
```diff
@@ -206,6 +216,22 @@
...
+.calendar:noPadding::body {
+    padding: 0;
+}
+
+.calendar:noPadding::weekdays {
+    padding: 0;
+}
+
+.calendar:noPadding::month {
+    min-height: auto;
```
Comment (sivanharel) [inline packages/wix-ui-tpa/src/components/DatePicker/DatePicker.st.css:232]:
```suggestion
    min-width: 240px;
    min-height: 284px;
```

File: packages/wix-ui-tpa/src/components/DatePicker/DatePicker.tsx [hunk 4]
```diff
@@ -69,6 +69,8 @@ export interface DatePickerProps extends TPAComponentProps {
   yearDropdownAriaLabel?: string;
   /** Identifies the element that labels the years dropdown in calendar header */
   yearDropdownAriaLabelledBy?: string;
+  /** If true, renders date picker without outer padding */
+  noPadding?: boolean;
```
Comment (sivanharel) [inline packages/wix-ui-tpa/src/components/DatePicker/DatePicker.tsx:73]: So we decided to call it `removeOutterPadding` :)

**Review by sivanharel — CHANGES_REQUESTED** [review]
Very nice :)
Just some small comments.
After you will fix them we will send the link for Michal to get approval for that.

**Review by sivanharel — APPROVED** [review]
Great!
So if Michal approves we could merge it.

**Pull request merged**